docs/jade-getting-started.md

```diff
@@ -333,68 +255,54 @@ export GOOGLE_ALLOWREUSEEXISTINGBUCKETS=true
 export HOST=localhost
 ```
 
-## 13. Repository Setup
+## 10. Repository Setup
 
 ### 1. Build, run and Unit Test `jade-data-repo`
 
-* Start postgres
 * Ensure docker is running
-* You may need to re-auth with vault every so often. Run `vault login -method=github token=$(cat ~/.gh_token)`
-* Run `./render-configs.sh` to pull secrets from vault
-* Refresh your Z-shell configuration by running `source ~/.zshrc`
-* Build the code and run the unit tests:
+* Auth as your broadinstitute.org to pull from Google Secrets Manager `gcloud auth login <you>@broadinstitute.org`
+* Run `./scripts/run-db start` to start the DB in a docker container
+* Run `./scripts/run local` to run TDR locally or `./scripts/run docker` to run TDR in a docker container
+* To Build the code and run the unit tests:
 
 ```
-./gradlew build           # build jade-data-repo and run unit tests
-./gradlew bootRun         # build jade-data-repo with Spring Boot features
-./gradlew check           # linters and unit tests
+./scripts/build project   # build jade-data-repo and run unit tests
+./scripts/run check       # linters and unit tests
```

Comment:
what is the difference between project and check? because doesn't the project need to be built in order to run tests and don't linters run when the tests run?